group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetMetadataManager.java

```diff
@@ -325,24 +325,12 @@ private Group validateOffsetCommit(
             }
```

Comment:
I kind of forget why we wanted to check `GroupIdNotFoundException`. I feel the current implementation does support the classic protocol member